persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -88,7 +88,7 @@
       @SubExpr(
           name = "select",
           definedAs =
-              "select star? column* writeTimeColumn? from (where* perPartitionLimit? limit? orderBy*) allowFiltering?"),
+              "select star? column* writeTimeColumn? count? max? min? sum? avg? from (where* perPartitionLimit? limit? orderBy*) allowFiltering?"),
```

Comment:
CQL allows multiple functions in the same select, AFAIK.

What do you think about a grammar like `(colium|funciton)*`, where `function` => `count|min|max|avg|sum`?

Reply:
... and for supporting aliases, how about this: `((column|function) as?)*`?

We could keep the last selector (column or function) in a field on `QueryBuilderImpl` so that the `as` method can access it.

Reply:
I am basing that on the existing `writeTimeColumn?` -> it had the alias support but this is not specified here.
Should it be? If yes, how it worked if the AS for `writeTime` was not present?

Reply:
Re: multiple function I meant something like `select sum(x), sum(y) from table`... as far as I understand current grammar does not allow expressing that via the `QueryBuilder`.

Reply:
Re: aliases, perhaps we do not need them for ordinary columns, but I thought it would be nice to extend that functionality to all selector kids just for the sake of completeness... totally optional, though.

Reply:
Ok, I see. I will implement that support for N same invocations.
However, I don't see what you mean by 
> We could keep the last selector (column or function) in a field on QueryBuilderImpl so that the as method can access it.
I think that having one `as()` method is not enough. We need to carry the alias for each function call. Therefore we need those aliases per column, right?

Reply:
`as` can be implemented differently, of course. I personally like the style of `builder.select().max("col1").as{"max1")` vs. `builder.select().max("col1", "max1")` but it's debatable, of course :)

With the former grammar we could keep a pointer to the last `Selector` object inside the `QueryBuilderImpl` and when `as(name)` is called, we'd simply redirect that call to `currentSelector.setAlias(name)` (or something like that)... This is just a rough sketch, feel free to use a different approach.

Reply:
fixed